src/Gomoku.hs

```diff
@@ -86,33 +86,27 @@ spottostr B = "B"
 boardtostr :: [[Tile]] -> [[String]]
 boardtostr (h:t) = map (map spottostr) (h:t)
 
--- Helper for printing, print the each row of the board
-printline [] =   
-  do
-      putStr ""
-printline (h:t) =
-  do
-      putStr (h ++ " ")
-      printline t
+-- Helper for Loading, convert character to type
+strtotile :: String -> Tile
+strtotile "O" = O
+strtotile "B" = B 
+strtotile "W" = W
 
--- Helper for printing, print the board representation in correct format
-printboard :: [[String]] -> IO ()
-printboard [] =   
-  do
-      putStr ""
-printboard (h:t) =
-  do
-      printline h
-      putStrLn ""
-      printboard t
+-- Helper for Loading, convert 2D string array to 2D tile array
+strtoboard :: [[String]] -> [[Tile]]
+strtoboard lst = map (map strtotile) lst
+
+-- Convert the board to a string that can be printed or written to file
+gametostr :: [[Tile]] -> String
+gametostr board = unlines (map unwords (boardtostr board))
```

Comment:
Can you change unwords to a custom unwords function so instead of adding one space between the tiles we will add two spaces. Something like this:
**myunwords :: [String] -> String
myunwords [] = []
myunwords (x:xs) = x ++ "  " ++ myunwords xs**
Just change the above function to work with 2 spaces this should be better since with 2 spaces the board actually looks like a square now instead of a rectangle.